components/eamxx/src/share/io/tests/io_basic.cpp

```diff
@@ -224,20 +244,42 @@ void read (const std::string& avg_type, const std::string& freq_units,
     reader.read_variables(n);
     for (const auto& fn : fnames) {
       auto f0 = fm0->get_field(fn).clone();
+      if (fn == "f_filled") {
+        f0 = fm0->get_field("f_3").clone();
+      }
       auto f  = fm->get_field(fn);
       if (avg_type=="MIN") {
         // The 1st snap in the avg window (the smallest)
         // is one past window_start=n*freq
         add(f0,n*freq+1);
+	// TODO: The MIN function ignores the initial condition, so the filling
+	// doesn't change any results for this test.  
+	// QUESTION: Should MIN include initial condition, should MAX as well?
```

Comment:
I don't think so. We do average (including min/max) by picking the rhs point of each interval.